tests/ModelTestStateTest.php

```diff
@@ -37,6 +37,28 @@ public function testInitialization()
         $this->assertEquals($originRecords, $state);
     }
 
+    public function testInitializationViaPrepareModelTestState()
+    {
+        $datasetMock = collect($this->getJsonFixture('initialization/dataset.json'));
+        $this->mockGettingDataset($datasetMock);
+
+        $mock = $this->getMockBuilder(TestCase::class)
+            ->disableOriginalConstructor()
+            ->getMockForAbstractClass();
+
+        $reflection = new ReflectionClass($mock);
+
+        $this->assertFalse($reflection->getProperty('globalExportMode')->getValue($mock));
+
+        $setGlobalExportMode = $reflection->getMethod('setGlobalExportMode');
+        $setGlobalExportMode->invoke($mock);
+
+        $prepareModelTestState = $reflection->getMethod('prepareModelTestState');
+        $prepareModelTestState->invoke($mock, TestModel::class);
+
+        $this->assertTrue($reflection->getProperty('globalExportMode')->getValue($mock));
```

Comment:
what this test case testing? you need to test that the global export mode property of created test state has the same value as in the TestCase class